README.md

```diff
@@ -1,72 +1,84 @@
-- **Coding Habits :owl:**: Gain insights into your coding habits, such as peak coding hours and most active days.
-- **Cobtribtuion Styles :evergreen_tree:**: Are you pushing your code, or are you stuck in Pull Request Reviews ?
-- **Catching fires before they spread :firefighter:**: Are you often requesting changes and finding bugs before they reach production ?
-- **And Much More :woman_astronaut:**
-
-<a href="https://unwrapped.dev"><img src="https://github.com/middlewarehq/unwrapped/assets/7949047/390e9d7a-183e-41c6-bcde-599d91df732f" width="100%" /></a>
-
-<h2 align='center'>🧑‍💻 Project Setup 🧑‍💻</h2>
-
-<h3>Environment Setup</h3>
-
- - Yarn Installation:
-    - Open a terminal and run the following commands:
-       ```
-       curl -o- -L https://yarnpkg.com/install.sh | bash
-       ```
-      Then, add Yarn to your shell configuration file (e.g., .bashrc or `.zshrc):
-      ```
-      export PATH="$HOME/.yarn/bin:$PATH"
-      ```
-- Node.js Installation:
-    - Use Node Version Manager (nvm) to install Node.js version 20.00:
-      ```
-       curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
-      ```
-      Rstart your terminal and install node.
-      ```
-      nvm install 20.10.0
-      nvm use 20.10.0
-      ```
-  
-<h3>CodeBase Setup</h3>
-
-- [Fork and Clone the Repository.](https://foundations.projectpythia.org/foundations/github/github-cloning-forking.html)
-- In the project repostory `/unwrapped` install dependancies.
-   ```
-   yarn install
-   ```
-- [Setup a Github Oauth App](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app) for `GITHUB_ID` and `GITHUB_SECRET`.
-- [Obtain Next Auth Secret](https://next-auth.js.org/configuration/options)
-- [Generating `TOKEN_ENC_PUB_KEY` and `TOKEN_ENC_PRI_KEY` RSA Key pair](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
-- Add a config file names `.env.local` with the following structure:
+<a href="https://unwrapped.dev"><img src="https://github.com/middlewarehq/year-in-review/assets/7949047/8984725a-dcd0-4568-9922-9e50db20313f)](https://unwrapped.dev" width="100%" align='center' /></a>
+
+<h1 align='center'>🚀 Features 🚀</h1>
+
+- **Code Contributions 🎖️**: View a summary of your contributions across various repositories over the past year.
+- **Productivity Insights 👩‍💻**: Discover time spent waiting for reviews and identify top contributors to your code reviews.
+- **Repo Highlights**: Showcase noteworthy projects you've worked on during the year.
+- **Coding Habits 🦉**: Gain insights into your coding patterns, including peak coding hours and most active days.
+- **Contribution Styles 🌳**: Understand your contribution habits—are you more focused on pushing code or stuck in pull request reviews?
+- **Early Bug Detection 🚒**: Identify trends in bug detection and code changes before they reach production.
+- **And Much More 🧑‍🚀**
```

Comment:
Might want to reduce the scale of the changes down a bit.
We definitely don't have anything to do with bug detection in this repo.

However, some of the languages changes are nice. Don't undo all your changes.

Some of the changes feel like changes made for the sake of increasing line count.